src/App.js

```diff
@@ -0,0 +1,47 @@
+import './App.css';
+import {Routes, Route, Navigate} from "react-router-dom";
+import {Home} from "./pages/Home";
+import {VideoDetails} from "./pages/VideoDetails";
```

Comment:
try to import these files using single index.js files it will increase readability